dask/dataframe/io/io.py

```diff
@@ -542,6 +543,9 @@ def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed"):
         If None, then won't use index information
     prefix : str, optional
         Prefix to prepend to the keys.
+    check_meta_flag : bool, optional
+        If False don't check that the incoming dataframe metadata are the 
+        same as the outbound metadata, defaults to True
```

Comment:
The terms inbound and outbound aren't immediately clear to me here.

How about "Check that the partitions have consistent metadata"